backend/inspirehep/mailing/views.py

```diff
@@ -38,3 +42,36 @@ def subscribe_jobs_weekly():
     except Exception:
         LOGGER.exception("Cannot subscribe user to list.", user=data)
         return jsonify({"message": "Unexpected error."}), 500
+
+
+@blueprint.route("/rss/jobs/weekly", methods=["GET"])
+def get_weekly_jobs_rss():
+    redis_url = current_app.config.get("CACHE_REDIS_URL")
+    weekly_jobs_email_key = current_app.config.get("WEEKLY_JOBS_EMAIL_REDIS_KEY")
+
+    redis = StrictRedis.from_url(redis_url)
+
+    [raw_title, raw_content, raw_timestamp] = redis.hmget(
+        weekly_jobs_email_key, ["title", "html_content", "timestamp"]
+    )
+
+    title = raw_title.decode("UTF-8")
+    content = raw_content.decode("UTF-8")
+    timestamp = float(raw_timestamp)
+    date = datetime.fromtimestamp(timestamp)
+
+    feed = AtomFeed(
+        title="Weekly HEP jobs - INSPIRE", feed_url=request.url, url=request.url_root
```

Comment:
feed `title` and feed entry `author` are hardcoded, still not sure if we should make them config vars, since I think they will not appear anywhere in the user email (TBC), therefore unlikely to be changed later.